Naming.java

```diff
@@ -0,0 +1,31 @@
+package az.alizeynalli.cleancode.naming;
+
+public class Naming {
+
+    public Date d;
+    public List<Testcase> find(User user){};
+    
+    public Account[] AccountList;
+
+    public void deleteUser(User pUser){};
+    
+    public URL r;
+  
+    public class TableRepresent{};
+    pubcli void car(){};
+  
+    public saveUser(){}; createAccount(){}; generateContract(){};
```

Comment:
// Pick one word for concept
    // Good
    public saveUser(){}; saveAccount(){}; saveContract(){};